src/mmap.cpp

```diff
@@ -384,37 +354,28 @@ mmap_reads(off_t partial_file_size,
         // the read tag
         reads[index++] = line;
         while ( *line++ != '\n' );
-                  
         // the read
         reads[index++] = line;
-                  
         // compute the read length
         while ( *line++ != '\n' ) readlen++;
-        if ( readlen >= 20 ) readlen < minlenread ? minlenread = readlen : minlenread;
-                  
+        if ( readlen >= 20 ) readlen < minlenread ? minlenread = readlen : minlenread; 
```

Comment:
20 has been replaced by the seed length (LNWIN). Added an inline comment to describe the line.